x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -208,7 +236,7 @@ function restructureConnectorsThatSupportIncident(
             subAction: 'pushToService',
             subActionParams: {
               incident: {
```

Comment:
Is there potential data loss on `externalId`, `category`, and `subcategory`? Part of the `CommonAttributes`.

Reply:
these fields `category` and `subcategory` have only being added in 7.12 and this migration is for 7.11.2. So we should be good here.

Reply:
'externalId' is only use from Case. it is not possible to have it there